index.html

```diff
@@ -0,0 +1,427 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>GamBit Store</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+    />
+    <link rel="stylesheet" href="https://gambitui.netlify.app/gambit.min.css" />
+    <link rel="stylesheet" href="/style.css" />
+  </head>
+
+  <body>
+    <header class="head">
+      <h1 class="logo"><a href="#">♞GamBit Store</a></h1>
+
+      <form action="get" class="header-search">
+        <input type="text" name="search-box" id="search" placeholder="Search" />
+        <button class="btn btn-primary" type="submit">
+          <i class="fas fa-search"></i>
+        </button>
+      </form>
+
+      <nav class="header-nav">
+        <ul>
+          <li>
+            <a href="#"><i class="fas fa-user-circle header-icon"></i>Login</a>
+          </li>
+          <li>
+            <a href="#"
+              ><i class="fas fa-solid fa-heart header-icon"></i>Whishlist
+            </a>
+          </li>
+          <li>
+            <a href="#"><i class="fas fa-shopping-cart header-icon"></i>Cart</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <aside class="category-list">
+      <div class="category-topic">
+        <a href="#" target="_blank"
+          ><i class="fa fa-regular fa-gift"></i>
+          <p>Top Offers</p></a
+        >
+      </div>
+      <div class="category-topic">
+        <a href="#" target="_blank"
+          ><i class="fa fa-regular fa-book"></i>
+          <p>Books</p></a
+        >
+      </div>
+      <div class="category-topic">
+        <a href="#" target="_blank"
+          ><i class="fa fa-regular fa-chess"></i>
+          <p>Chess Set</p></a
+        >
+      </div>
+      <div class="category-topic">
+        <a href="#" target="_blank">
+          <i class="fa fa-regular fa-book-open"></i>
+          <p>Note Books</p></a
+        >
+      </div>
+      <div class="category-topic">
+        <a href="#" target="_blank">
+          <i class="fa fa-regular fa-pen"></i>
+          <p>Pens</p></a
+        >
+      </div>
+    </aside>
+
+    <main class="main-store">
+      <div class="hero-container">
+        <div class="hero-head">
+          <h1 class="head-1">Exclusive Chess Products</h1>
+          <a class="btn btn-lg" href="#" role="button">Shop Now</a>
+        </div>
+      </div>
+
+      <div>
+        <h2 class="head-2 store-head">Deals of the Day</h2>
+      </div>
+
+      <div class="grid grid-col grid-gap">
```

Comment:
It would be better to use 2-3 variables for grid gap (like `.grid-gap-sm, .grid-gap-md, .grid-gap-lg` etc.) of different sizes, than just a single constant value

Reply:
yes, I will change that to a variable. Thank You.